rules-tests/Php74/Rector/Closure/ClosureToArrowFunctionRector/Fixture/skip_arrow_function_already_with_comment.php.inc

```diff
@@ -0,0 +1,9 @@
+<?php
+
+fn() =>
+    /**
+     * comment
+     */
+    // something
+    // @psalm-suppress UndefinedFunction
+    ff();
```

Comment:
this tweak on `BetterStandardPrinter::p()` currently make duplicate comment for existing arrow function:

```diff
      */
     // something
     // @psalm-suppress UndefinedFunction
-    ff();
+    /**
+     * comment
+     */
+// something
+// @psalm-suppress UndefinedFunction
```

which incorrect.